src/main/java/ua/antibyte/analyzer/service/impl/FileCsvParser.java

```diff
@@ -0,0 +1,61 @@
+package ua.antibyte.analyzer.service.impl;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+import org.springframework.stereotype.Service;
+import ua.antibyte.analyzer.dto.CommentDto;
+import ua.antibyte.analyzer.service.FileParser;
+
+@Service
+public class FileCsvParser implements FileParser<CommentDto> {
+    private static final String[] FILE_HEADER = new String[]{
+            "Id", "ProductId", "UserId", "ProfileName", "HelpfulnessNumerator",
+            "HelpfulnessDenominator", "Score", "Time", "Summary", "Text"};
+
+    @Override
+    public List<CommentDto> parse(String path) {
+        try {
+            CSVParser csvParser = getCsvParser(Files.newBufferedReader(Paths.get(path)));
+            List<CommentDto> commentDtos = new ArrayList<>();
+            for (CSVRecord csvRecord : csvParser) {
+                if (FILE_HEADER[0].equals(csvRecord.get(FILE_HEADER[0]))) {
```

Comment:
can we just skip the first line?
or check the first record separately, so that this check won't be repeated for all the records?